lib/packwerk/association_inspector.rb

```diff
@@ -22,20 +22,20 @@ def initialize(inflector: Inflector.new, custom_associations: Set.new)
     end
 
     def constant_name_from_node(node, ancestors:)
-      return unless Node.type(node) == Node::METHOD_CALL
+      return unless Node.method_call?(node)
```

Comment:
My [comment on #39](https://github.com/Shopify/packwerk/pull/39#discussion_r513322686) delves into this a bit. If we can make these predicates the only type-related API that `Node::Packwerk` exposes then I’m in favour, but I’m unsure about adding them while also keeping `Node.type`.

Reply:
I might be projecting, but I think you are raising a critical point: the Node module's API is a bit messy. To your point, let's make these predicates the only type-related API that `Node` exposes. I want to do that change in another PR since it seems a bit more practical.